tests/integration/targets/grafana_datasource/tasks/thruk.yml

```diff
@@ -0,0 +1,49 @@
+- name: Create thruk datasource
+  register: result
+  grafana_datasource:
+    name: datasource-thruk
+    grafana_url: "{{ grafana_url }}"
+    grafana_user: "{{ grafana_username }}"
+    grafana_password: "{{ grafana_password }}"
+    org_id: "1"
+    ds_type: sni-thruk-datasource
+    url: "https://thruk.company.com/sitename/thruk"
+    tls_skip_verify : yes
+    validate_certs: no
+- debug:
```

Comment:
please remove the debug task here.

Reply:
Please also add an empty line between tasks to ease readability.

Reply:
> please remove the debug task here.

Should I remove also the debug task in idempotency one?

Reply:
yep, the debug tasks should not remain in the tests roles.